vilebot/src/main/java/com/oldterns/vilebot/Vilebot.java

```diff
@@ -78,7 +116,7 @@ public static void main( String[] args )
+                                                                                                                         ircPort ).addAutoJoinChannel( ircChannel ).setAutoReconnect( true ).addListener( new Vilebot() ).addListener( new AdminManagement() ).addListener( new AdminPing() ).addListener( new Auth() ).addListener( new GetLog() ).addListener( new com.oldterns.vilebot.handlers.admin.Help() ).addListener( new NickChange() ).addListener( new com.oldterns.vilebot.handlers.admin.Ops() ).addListener( new Quit() ).addListener( new AnswerQuestion() ).addListener( new Ascii() ).addListener( new ChatLogger() ).addListener( new Church() ).addListener( new Countdown() ).addListener( new Decide() ).addListener( new Excuses() ).addListener( new FakeNews() ).addListener( new Fortune() ).addListener( new GetInfoOn() ).addListener( new Help() ).addListener( new ImageToAscii() ).addListener( new Inspiration() ).addListener( new Jaziz() ).addListener( new Jokes() ).addListener( new Kaomoji() ).addListener( new Karma() ).addListener( new KarmaRoll() ).addListener( new LastMessageSed() ).addListener( new LastSeen() ).addListener( new Markov() ).addListener( new News() ).addListener( new Omgword() ).addListener( new Ops() ).addListener( new QuotesAndFacts() ).addListener( new Reddit() ).addListener( new RemindMe() ).addListener( new RockPaperScissors() ).addListener( new Trivia() ).addListener( new Ttc() ).addListener( new TwitterCorrection() ).addListener( new UrlTitleAnnouncer() ).addListener( new UrlTweetAnnouncer() ).addListener( new Userlists() ).addListener( new UserPing() ).addListener( new Weather() ).buildConfiguration();
```

Comment:
Y'all, **what** is this line? Ends on column 1645?? At the very least break it up a little:

```java
            Configuration botConfiguration =
                new Configuration.Builder()
                    .setName( ircNick )
                    .setLogin( ircUser )
                    .setRealName( ircRealName )
                    .addServer( ircServerAddress, ircPort )
                    .addAutoJoinChannel( ircChannel )
                    .setAutoReconnect( true )
                    .addListener( new Vilebot() )
                    .addListener( new AdminManagement() )
                    .addListener( new AdminPing() )
                    .addListener( new Auth() )
                    .addListener( new GetLog() )
                    .addListener( new com.oldterns.vilebot.handlers.admin.Help() )
                    .addListener( new NickChange() )
                    .addListener( new com.oldterns.vilebot.handlers.admin.Ops() )
                    .addListener( new Quit() )
                    .addListener( new AnswerQuestion() )
                    .addListener( new Ascii() )
                    .addListener( new ChatLogger() )
                    .addListener( new Church() )
                    .addListener( new Countdown() )
                    .addListener( new Decide() )
                    .addListener( new Excuses() )
                    .addListener( new FakeNews() )
                    .addListener( new Fortune() )
                    .addListener( new GetInfoOn() )
                    .addListener( new Help() )
                    .addListener( new ImageToAscii() )
                    .addListener( new Inspiration() )
                    .addListener( new Jaziz() )
                    .addListener( new Jokes() )
                    .addListener( new Kaomoji() )
                    .addListener( new Karma() )
                    .addListener( new KarmaRoll() )
                    .addListener( new LastMessageSed() )
                    .addListener( new LastSeen() )
                    .addListener( new Markov() )
                    .addListener( new News() )
                    .addListener( new Omgword() )
                    .addListener( new Ops() )
                    .addListener( new QuotesAndFacts() )
                    .addListener( new Reddit() )
                    .addListener( new RemindMe() )
                    .addListener( new RockPaperScissors() )
                    .addListener( new Trivia() )
                    .addListener( new Ttc() )
                    .addListener( new TwitterCorrection() )
                    .addListener( new UrlTitleAnnouncer() )
                    .addListener( new UrlTweetAnnouncer() )
                    .addListener( new Userlists() )
                    .addListener( new UserPing() )
                    .addListener( new Weather() )
                    .buildConfiguration();
```

Reply:
The VIleBot1 formatter does that. However, this task should be done in a more generic way.

Reply:
I thought it was a running joke tbh, which is why I never bothered to reformat it.

Reply:
Running jokes have no place in code, not even in joke code

Reply:
Then the formatter is busted. Alternatively, 

```java
Builder builder = new Configuration.Builder();
builder.setName( ircNick );
builder.setLogin( ircUser )
// [...]
```